integrationExamples/gpt/userId_example.html

```diff
@@ -215,6 +215,13 @@
                         name: "_li_pbid",
                         expires: 28
                       }
+                    },
+                    {
+                      name: "zeotapIdPlus",
+                      storage: {
+                        type: "cookie",
+                        name: "IDP"
+                      }
```

Comment:
it looks like you are defining your own name for the cookie and you're not leveraging the userId module storage (which is what this bit is used for). so I think you can just ignore this, like how Parrable does.

based on your module code, I think you can just do this:

```suggestion
                    },
                    {
                      name: "zeotapIdPlus"
```

Reply:
Hey @smenzer, thanks for the suggestion, have made changes to the module and files to remove storage configuration.